contentcuration/contentcuration/frontend/shared/leUtils/FormatPresets.js

```diff
@@ -464,6 +464,22 @@ const FormatPresetsMap = new Map([
       associated_mimetypes: ['image/jpeg', 'image/png'],
     },
   ],
+  [
```

Comment:
Did you run this code generation command, or did you do this manually? https://github.com/learningequality/studio/blob/unstable/deploy/generatejsconstantfiles.py

Would be good to run this if not, and commit the results (but make sure to run linting when you do) - just to make sure we've got everything covered.